core/src/hash_table/header.rs

```diff
@@ -44,49 +47,39 @@ impl Header {
             entry_type: entry.entry_type().clone(),
             // @TODO implement timestamps
             // https://github.com/holochain/holochain-rust/issues/70
-            time: String::new(),
-            next: chain.top().as_ref().map(|p| p.header().hash()),
-            entry: entry.hash().to_string(),
-            type_next: chain
+            timestamp: String::new(),
+            prev: chain.top().as_ref().map(|p| p.header().hash()),
+            entry_hash: entry.hash().to_string(),
+            prev_same: chain
                 .top_type(&entry.entry_type())
                 // @TODO inappropriate expect()?
                 // @see https://github.com/holochain/holochain-rust/issues/147
                 .expect("top type should never error")
                 .map(|p| p.header().hash()),
             // @TODO implement signatures
             // https://github.com/holochain/holochain-rust/issues/71
-            signature: String::new(),
+            entry_signature: String::new(),
         }
     }
 
-    /// entry_type getter
+    /// getters
```

Comment:
this isn't going to play nice with the generated API docs is it?

i have been adding a comment to every function individually as each one has it's own page/section in the generated docs